src/LaravelRequestDocs.php

```diff
@@ -133,18 +134,16 @@ public function appendRequestRules(array $controllersInfo)
                     //throw $th;
                 }
 
-                if ($requestClass && method_exists($requestClass, 'rules')) {
-                    try {
-                        $controllersInfo[$index]['rules'] = $this->flattenRules($requestClass->rules());
-                    } catch (Throwable $e) {
-                        // disabled. This only works when the rules are defined as 'required|integer' and that too in single line
-                        // doesn't work well when the same rule is defined as array ['required', 'integer'] or in multiple lines such as
-                        // If your rules are not populated using this library, then fix your rule to only throw validation errors and not throw exceptions
-                        // such as 404, 500 inside the request class.
-                        $controllersInfo[$index]['rules'] = $this->rulesByRegex($requestClassName);
+                foreach (config('request-docs.request_methods') as $rquestMethod) {
```

Comment:
I don't think that `config('request-docs.request_methods')` should be in the config file, there is just one method called rules that has all the rules.

Reply:
but in my case i use multiple methods for validation and in rules method only check some thing and call them example i use UserRequest.php in rules() method i check if the request is put if its put i will call onUpdate() method otherwise i will call onCreate() method

Reply:
Oh ok. Thanks for the explanation. I ll merge it soon by this week